sosw/components/helpers.py

```diff
@@ -887,3 +890,33 @@ def to_bool(val):
         elif val.lower() in ['false', '0']:
             return False
     raise Exception(f"Can't convert unexpected value to bool: {val}, type: {type(val)}")
+
+
+def get_message_dict_from_sns_event(event):
+    """
+    Extract SNS event message and returns it as dict
+
+    :param dict event: Lambda SNS event (payload)
+
+    :rtype dict
+    :return: The SNS message, converted to dict
+    """
+
+    message = event['Records'][0]['Sns'].get('Message')
+
+    return json.loads(message)
```

Comment:
json.loads(None) - raise
json.loads("") - raise...
pretty unstable behaviour. You probably want to catch these ones at least are return empty dict.